src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,30 @@
+import './_checkbox.scss';
+import { CheckboxProps } from '../../types/components/componentType';
+
+const Checkbox: React.FC<CheckboxProps> = ({
+  className,
+  classNameText,
+  checkboxText,
+  checked,
+  onChange,
+}) => {
+  const checkboxId =
+    'checkbox-' + checkboxText.toLowerCase().replace(/\s/g, '-');
+
+  return (
+    <div className="checkbox-container">
+      <input
+        className={className}
+        id={checkboxId}
+        type="checkbox"
+        checked={checked}
+        onChange={onChange}
+      />
+      <label className="checkbox-label" htmlFor={checkboxId}>
+        <p className={classNameText}>{checkboxText}</p>
```

Comment:
Instead of using a `'<p>'` element for the checkbox text, you might consider using a more semantic element like `'<span>'` or `'<div>'.` The choice depends on your specific styling needs, but it's generally better to reserve `'<p>'` for paragraphs.

Reply:
Minor*